playground/ssr-vue/__tests__/ssr-vue.spec.ts

```diff
@@ -26,7 +27,8 @@ test('vuex can be import succeed by named import', async () => {
 })
 
 test('/about', async () => {
-  await page.goto(url + 'about')
+  await untilBrowserLogAfter(() => page.goto(url + 'about'), 'hydrated')
+
   expect(await page.textContent('h1')).toMatch('About')
   // should not have hydration mismatch
   browserLogs.forEach((msg) => {
```

Comment:
Also added here because there's a test checking hydration mismatch that requires hydration to be completed.